pkg/util/net/ip/ip_utils.go

```diff
@@ -0,0 +1,67 @@
+package util
+
+import (
+	"bufio"
+	"net"
+	"os"
+)
+
+const DISABLE_IPV6_PATH = "/proc/sys/net/ipv6/conf/default/disable_ipv6"
+const IPV4_BIND_ADDRESS = "0.0.0.0"
+const IPV6_BIND_ADDRESS = "[::]"
+const IPV4_LOOPBACK_ADDRESS = "127.0.0.1"
+const IPV6_LOOPBACK_ADDRESS = "[::1]"
+
+// GetIPBindAddress returns IP bind address (either 0.0.0.0 or [::] according sysctl disable_ipv6)
+func GetIPBindAddress() string {
```

Comment:
Does 'IPBind' id some known terminology to "0.0.0.0"? (I found INADDR_ANY)

Reply:
I think this was suggestion of Miguel
indeed the define is right INADDR_ANY
but using GetIPAnyAddress and so on is confusing, this what Miguel said once